otx/algorithms/segmentation/adapters/mmseg/datasets/pipelines/compose.py

```diff
@@ -11,8 +13,11 @@
 from scipy.ndimage import gaussian_filter
 
 
+# pylint: disable=consider-using-f-string
```

Comment:
This is an issue that is resolved by modifying strings using `.format` to `f-string`, so it seems like a good to apply it.

Reply:
Revised. Thanks:)